### DISCUSSION TITLE
HTTP API

### DISCUSSION THREAD

**Discussion by kenwiesner** [raspicamplayer/camplayer#31] [Ideas]
An API over HTTP would be great for the basic remote commands! This way if you have a home automation system you can integrate camplayer with your remote system.

http://camplayer/device/X/channel/X
http://camplayer/window/1
etc...

**Comment by donkurt** [comment]
This would be fantastic!  Suuuuper impressed with Camplayer, but missing an API to interact with (and only allowing keyboard commands) is a limiting factor for full integration into my smart home.

**Comment by OtherwisePrickly** [comment]
+1 on this feature. I tied using xdotool to send key strokes over ssh. The raspberry pi receives the commands but for some reason they work in the background and not in Camplayer. An API would really help.
